tests/skip_inner_frames.rs

```diff
@@ -23,7 +23,6 @@ fn backtrace_new_unresolved_should_start_with_call_site_trace() {
     assert!(!b.frames().is_empty());
 
     let this_ip = backtrace_new_unresolved_should_start_with_call_site_trace as usize;
-    println!("this_ip: {:?}", this_ip as *const usize);
```

Comment:
So why did you remove these assertions?

If this is because `:p` internally calls `expose_addr`, then we should probably start the discussion of using `addr` there instead -- and in the mean time either gate these prints by `cfg(not(miri))` or just remove the `-Zmiri-strict-provenance` from the CI flags.

Reply:
Yes, this is because `:p` internally calls `expose_addr`.

If everyone else is comfortable with it, I think the most appropriate approach is to remove `-Zmiri-strict-provenance`, because as i said above, this codebase plays a bit fast and loose with types, and it wouldn't hurt to address that... but not here.